frontend/src/vault/share/ShareVaultCard.tsx

```diff
@@ -48,14 +52,20 @@ const LogoContainer = styled.div`
 export const ShareVaultCard = () => {
   const vault = useAssertCurrentVault();
 
-  const { name, public_key_ecdsa, public_key_eddsa } = vault;
+  const { name, local_party_id } = vault;
 
   const qrCodeValue = JSON.stringify(getVaultPublicKeyExport(vault));
 
   const { colors } = useTheme();
 
   return (
     <Container>
+      <VStack gap={4} alignItems="center">
+        <Text weight={600} size={22} cropped>
+          {name}
+        </Text>
+        <VaultKey title="UID" value={local_party_id} />
```

Comment:
UID , is not local party id , it should be https://github.com/vultisig/vultisig-windows/blob/main/frontend/src/vault/share/utils/getVaultPublicKeyEport.ts#L19